man/patterns.Rd

```diff
@@ -12,14 +12,15 @@ and melt them into separate columns. See the \code{Efficient reshaping using
 data.tables} vignette linked below to learn more.
 }
 \usage{
-patterns(\dots, cols=character(0))
+patterns(\dots, cols=character(0), perl=FALSE)
 }
 \arguments{
   \item{\dots}{A set of regular expression patterns.}
   \item{cols}{A character vector of names to which each pattern is matched.}
+  \item{perl}{Passed to \code{grep}: \code{TRUE} to use PCRE, \code{FALSE} to use TRE.}
```

Comment:
```suggestion
  \item{perl}{Passed to \code{\link{grep}}.}
```

Probably better to just point to `?grep`; R users that don't know what `perl=` does yet are unlikely to understand "PCRE" or "TRE" either, I reckon.